scripts/build/macOS/build_universal.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+buildDirectory=$1
+buildType=$2
+
+scriptLocation=$(dirname "$(readlink -f "$0")")
+sourceLocation=$(readlink -f "$scriptLocation/../../..")
+
+cmake -B "${buildDirectory}/x64" -S "${sourceLocation}" -DCMAKE_BUILD_TYPE="${buildType}" -DCMAKE_CONFIGURATION_TYPES="${buildType}" -DMACOS_ARCHITECTURE=x86_64 -G Xcode
+cmake --build "${buildDirectory}/x64" --config "${buildType}"
+
+cmake -B "${buildDirectory}/arm64" -S "${sourceLocation}" -DCMAKE_BUILD_TYPE="${buildType}" -DCMAKE_CONFIGURATION_TYPES="${buildType}" -DMACOS_ARCHITECTURE=arm64 -G Xcode -DIMAGE_COMPILER_EXECUTABLE="${buildDirectory}/x64/utils/${buildType}/image-compiler"
```

Comment:
Why was the `-DIMAGE_COMPILER_EXECUTABLE` not given on line 11 too?

Reply:
Line 11 builds image-compiler, line 14 does not